.github/release.yaml

```diff
@@ -28,4 +28,7 @@ changelog:
         - devops
     - title: Other Changes 🧑‍💻
       labels:
-        - "*"
\ No newline at end of file
+        - "*"
+    - title: Dependency Upgrades 🆙
+        labels:
+          - dependencies
```

Comment:
Just to make the flow make more sense, please put this block above the "Other Changes" section - that's the final fallback if someone forgot to label their PR.